index.js

```
@@ -182,10 +287,18 @@ function NEXT_VESTING_DATE(startDate, schedule) {
  * @note This is a placeholder implementation and should be replaced with 
  * more sophisticated vesting calculation logic in a production environment.
  */
-function NEXT_VESTING_AMOUNT(totalShares, schedule) {
-    // This would normally calculate the next vesting amount
-    // For demonstration, returning 1/16th of total shares (quarterly vesting)
-    return totalShares / 16;
+function get_next_vesting_amount(totalShares, schedule = 'quarterly') {
+    if (!totalShares || isNaN(totalShares)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)